platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

```diff
@@ -90,16 +109,32 @@ This procedure describes how to configure Data Studios for Kubernetes deployment
       INITIAL_ACCESS_TOKEN: <INITIAL_ACCESS_TOKEN>
     ```
 
-1. Edit your ConfigMap named `platform-backend-cfg` for Platform and add the following environment variables:
+1. Edit the ConfigMap named `platform-backend-cfg` in the `configmap.yml` for Platform.
+    -  Add the following environment variables:
+
+       - `TOWER_DATA_STUDIO_CONNECT_URL`
+       - `TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN`: Use the same access token that you specified in the previous step
+       - `TOWER_OIDC_PEM_PATH`
 
-  ```yaml
-    TOWER_DATA_STUDIO_ALLOWED_WORKSPACES=<ALLOWED WORKSPACES>
-    TOWER_DATA_STUDIO_CONNECT_URL=<YOUR CONNECT URL, SAME AS `CONNECT_PROXY_URL` IN `data-studios.env`>
-    TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN=<YOUR INITIAL ACCESS TOKEN, IDENTICAL AS `INITIAL_ACCESS_TOKEN` in `data-studios.env`>
-    TOWER_OIDC_PEM_PATH=<PATH TO PEM FILE>
-  ```
+    - If you want to restrict Data Studios to specific organization workspaces, list each workspace ID in the `TOWER_DATA_STUDIO_ALLOWED_WORKSPACES`.
+    - If you want to allow access to Data Studios in every organizational workspace, do not include the `TOWER_DATA_STUDIO_ALLOWED_WORKSPACES`.
 
-1. Apply your updated configuration: `kubectl apply -f configmap.yml`
+1. Optional: If you want to allow access to Data Studios in every organizational workspace, edit the ConfigMap named `tower-yml` in the `configmap.yml` and include the following stanza with your existing configuration:
+
+    ```yaml
+    data:
+      tower.yml: |-
+        tower:
+          data-studio:
+            allowed-workspaces: null
+    ```
+
+1. Apply the updated configuration: `kubectl apply -f configmap.yml`
+1. Apply the Data Studios manifests:
+
+    ```
+    kubectl apply -f ingress.aks.yml proxy-config.yml proxy.yml server.yml
```

Comment:
why `aks`?

Reply:
@gwright99, this seems to be K8s provider specific, and this one appears to be for AWS, the only supported public cloud Data Studios platform today.